composer.json

```diff
@@ -39,15 +39,16 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "illuminate/support": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/console": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "doctrine/dbal": "^2.6.0"
+        "php": "^7.2",
+        "illuminate/support": "~6.0",
+        "illuminate/console": "~6.0",
+        "doctrine/dbal": "^2.6.0",
+        "laravel/helpers": "~1.0"
```

Comment:
I would recommend to do not use helpers also in this packages and definitely not in generated files.

Reply:
I tried to not use laravel/helpers in any production package. generator is dev dependency, so I find it okay to use it here. 

Generated code should not use helpers, only generator itself may use it. At least that was my intention. What do you think?

Reply:
Ok